src/components/MakeWidget/subcomponents/RateField/RateField.styles.tsx

```diff
@@ -0,0 +1,34 @@
+import styled, { css } from "styled-components/macro";
+
+import { fontMono } from "../../../../style/themes";
+
+export const RateFieldStyle = css`
+  color: ${({ theme }) => theme.colors.lightGrey};
+  text-transform: uppercase;
+  font-size: 0.75rem;
+`;
+
+export const Text = styled.div`
+  margin-top: -2px;
+`;
+
+export const Wrapper = styled.div`
+  ${RateFieldStyle};
+  display: flex;
+  align-items: center;
+  gap: 0.375rem;
+  font-weight: bold;
+
+  & > .icon {
+    border: none;
+  }
```

Comment:
```suggestion
```

The border should stay for focus and active states.

Reply:
The default border styling was doing some weird stuff on active. I will add hover and active borders with the themes (grey/blue).